var/spack/repos/builtin/packages/vim/package.py

```diff
@@ -42,11 +42,14 @@ class Vim(AutotoolsPackage):
     variant('python', default=False, description="build with Python")
     variant('ruby', default=False, description="build with Ruby")
     variant('x', default=False, description="use the X Window System")
+    variant('gtk3', default=False, description="use the GTKv3 gui.")
```

Comment:
```suggestion
    variant('gtk', default=False, when='+gui', description="use the GTKv3 gui.")
```
Change variant name for consistency, and only apply when GUI option is enabled

Reply:
thanks for the suggestion